calc/src/mechanics/gen789.ts

```diff
@@ -3095,6 +3133,12 @@ export function calculateFinalModsSMSSSV(
     finalMods.push(8192);
   }
 
+  if(defender.name.includes('Flareon') && (field.chromaticField === 'Rainbow')) {
+    if (defender.hasStatus('brn') || defender.hasStatus('psn') || defender.hasStatus('tox')) {
+      desc.chromaticField = field.chromaticField;
+    }
+  }
```

Comment:
1- Just wondering, why did you put this code here specifically?
2- suggestion:
```suggestion
  if(defender.name.includes('Flareon') && (field.chromaticField === 'Rainbow')) {
    if (defender.hasStatus('brn', 'psn', 'tox')) {
      desc.chromaticField = field.chromaticField;
    }
  }
```

Reply:
ya this comma thing is something I should have realized was built in

Reply:
otherwise I put it here because it had to go somewhere, and I was nervous to touch the existing magic guard code

Reply:
search for `// Fields - Text for description`, that is where I put all these "ugly" solyutions for printing to the description

Reply:
moved my field descriptions there, listed below underwater. flareon, umbreon and sylveon